cql2/standard/clause_7_enhanced.adoc

```diff
@@ -208,15 +208,33 @@ The spatial data types are (literal rule `spatialLiteral`):
 * "Polygon": a planar surface bounded by closed line strings;
 * "MultiPolygon": a collection of polygons;
 * "Geometry": an instance of "Point", "MultiPoint", "LineString", "MultiLineString", "Polygon", or "MultiPolygon";
-* "GeometryCollection": a collection of "Geometry" instances.
+* "GeometryCollection": a collection of "Geometry" instances;
+* "BBox": a bounding rectangle or box.
 
-Existing representations are used for literal values:
+Existing representations are in general used for literal values:
 
-* Text: an https://portal.ogc.org/files/?artifact_id=25355[OGC Well-Known Text (WKT)] literal
-* JSON: a https://www.rfc-editor.org/rfc/rfc7946.html#section-3.1[GeoJSON geometry] object
+* Text: an OGC Well-Known Text (WKT) literal (see clause 7 of <<ogc06-103r4,Simple feature access - Part 1: Common architecture>>)
+* JSON: a GeoJSON geometry object (see clause 3.1 of <<GeoJSON,GeoJSON>>)
+
+The BBox data type is an exception as it is not specified in WKT. In the Text representation, the type is encoded as a `BBOX()` operator with four or six numerical arguments, depending on whether the coordinates include a vertical axis (height or depth):
```

Comment:
Should this perhaps say `BBOX()` **_literal_** rather than _operator_?

In CQL2 and WKT, `( )` are used everywhere so _grammatically_ speaking this is all the _function call_ syntax (that is the main point of #705), but conceptually there is a distinction between actual execution of a function / operator and a literal definition. e.g., in [CMSS](https://docs.ogc.org/per/18-025.html#_expressions), we use `( )` for function calls, `[ ]` for arrays  and `{ }` for type instantiation.

(related to #718 suggesting to use `( )` everywhere in CQL2 for array literals to be consistent with WKT and `IN` syntax, which really are already using a kind of array literals defined between `( )`).

Reply:
Meeting 2022-08-01: @cportele to change to 'constructor' (and use this consistently with other cases like INTERVAL). Allowing properties in geometries is out-of-scope for this version. We might discuss this in a future version in the context of the future grammar simplification discussion.

Reply:
Change applied in https://github.com/opengeospatial/ogcapi-features/pull/745/commits/d5527b9282c91a7322cfab135fcbc50a5f5ccbe2